homework-2/src/components/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import MenuItem from "./MenuItem";
+
```

Comment:
```suggestion
import PropTypes from 'prop-types'
```
так как вы тут используете PropTypes